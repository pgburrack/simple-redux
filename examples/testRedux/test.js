var actions = require('./actions/publishers');

describe('actions', () => { // TODO: Add tests
  // [undefined, null, ''].forEach((param) =>
  // it(`should set publishers with default parameter ${param}` , () => {
  //    expect(actions.addRecipe(param)).toMatchSnapshot()
  // }));

  it('should create an action to set publishers', () => {
    const publishers = ['publisher1', 'publisher2'];

    expect(actions.setPublishers(publishers)).toMatchSnapshot();
  });
});