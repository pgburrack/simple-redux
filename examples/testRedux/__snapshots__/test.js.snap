// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions should create an action to set publishers 1`] = `
Object {
  "payload": Array [
    "publisher1",
    "publisher2",
  ],
  "type": "SET_PUBLISHERS",
}
`;
